ui/app/common/components/pnc-admin-panel/pncAdminPanel.js

```diff
@@ -79,6 +82,16 @@
       }
     };
 
+    $ctrl.editPNCVersion = function () {
+      $ctrl.edittingVersion = !$ctrl.edittingVersion;
+    }
+
+    $ctrl.updatePNCVersion = function () {
+      console.log($ctrl.pncVersion)
```

Comment:
please, avoid keeping `console.log` if there is no specific reason to leave it there

Reply:
Sorry, unnoticed leftover